tools/sdk-readme-table-generator/TableGenerator/Program.fs

```diff
@@ -8,25 +8,32 @@ open TableGenerator.Table
 let inputBranches =
         [ { GitBranchName = "master"
             DisplayName = "Master<br>(6.0.x&nbsp;Runtime)"
-            AkaMsChannel = Some("6.0/daily") }
+            AkaMsChannel = Some("6.0/daily") 
+            UseLegacyAkaMsTemplate = false }
```

Comment:
instead of making another property people have to consider everytime a new branch get created. Could you use parsed branch name and cap them to a certain version? For example if branch.Major = 5, patch < 300 use LegacyAkaMsTemplate.

Reply:
Done. This is my first real time doing any F#. Please let me know if the syntax isn't quite right.